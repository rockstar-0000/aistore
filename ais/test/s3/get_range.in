// SKIP "aws command does not handle redirects correctly"

printf "0123456789" > $OBJECT.txt
aws --endpoint-url $HOST s3 mb s3://$BUCKET // IGNORE
aws --endpoint-url $HOST s3 cp $OBJECT.txt s3://$BUCKET$OBJECT // IGNORE
aws --endpoint-url $HOST s3api get-object --bucket $BUCKET --key $OBJECT --range bytes=1-5 /tmp/objpart // IGNORE
cat /tmp/objpart
aws --endpoint-url $HOST s3api get-object --bucket $BUCKET --key $OBJECT --range bytes=4- /tmp/objpart // IGNORE
cat /tmp/objpart
aws --endpoint-url $HOST s3api get-object --bucket $BUCKET --key $OBJECT --range bytes=6-15 /tmp/objpart // IGNORE
cat /tmp/objpart
aws --endpoint-url $HOST s3api get-object --bucket $BUCKET --key $OBJECT --range bytes=-7 /tmp/objpart // IGNORE
cat /tmp/objpart
aws --endpoint-url $HOST s3 rb s3://$BUCKET // IGNORE
rm $OBJECT.txt // IGNORE
rm /tmp/objpart // IGNORE
