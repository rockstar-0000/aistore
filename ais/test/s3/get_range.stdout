12345
456789
6789
3456789
